db/db-postgresql/index.js

```diff
@@ -0,0 +1,15 @@
+const Sequelize = require('sequelize');
+
+const db = new Sequelize('api', 'me', '', {
+  host: 'localhost',
+  dialect: 'postgres',
+});
+
+
+db
+  .authenticate()
```

Comment:
Might be better if the .authenticate call was attached to the Sequelize instance here